woocommerce/woocommerce-ios
# [Mobile Payments] [Customers] Add a remote to create a wcpay customer on an order

Partially addresses #4298 

This PR adds customer creation to the WCPay Remote, taking advantage of the new create_customer endpoint added in plugin verison 2.8.3. A subsequent PR will integrate this into CardPresentPaymentStore to allow us to pass the customer ID into payment intent creation

To test:
- Run the new unit tests `test_fetchOrderCustomer_properly_returns_customer` and `test_fetchOrderCustomer_properly_handles_error_response` and ensure they pass

See also:
- https://github.com/Automattic/woocommerce-payments/pull/2501

Update release notes:
